CMakeLists.txt

```diff
@@ -84,7 +92,18 @@ macro(build_libyaml)
   set(yaml_LIBRARIES yaml)
 endmacro()
 
-build_libyaml()
+# Skip building yaml if the expected version is already present in the system
+if (yaml_FOUND)
+  if("${yaml_VERSION}" VERSION_EQUAL 0.2.5)
+    set(_SKIP_YAML_BUILD 1)
+  else()
+    message(WARNING "A wrong version of libyaml is already present in the system: ${yaml_VERSION}")
```

Comment:
@alsora shall we add that it will be ignored?

Reply:
Done